### PR TITLE
Fix remaining descriptions

### PR DESCRIPTION
**Pull request by larswik**
Only the ones related to IFC Tunnel.

### CODE REVIEW

File: IFC4x3/Sections/Resource definition data schemas/Schemas/IfcExternalReferenceResource/Entities/IfcDatasetInformation/DocEntity.xml [hunk 1]
```diff
@@ -4,7 +4,9 @@
 		<DocAttribute Name="SchemaReference" UniqueId="53e19df0-e203-4029-bdcd-56df644876da" DefinedType="IfcURIReference" AttributeFlags="1">
 			<Documentation>An optional attribute for referencing a schema that defines the possible content and structure of the dataset (e.g. xsd).</Documentation>
 		</DocAttribute>
-		<DocAttribute Name="DatasetInfoForObjects" UniqueId="b516123d-f0a9-4d8d-b03f-d43496183c16" DefinedType="IfcRelAssociatesDataset" AggregationType="3" Inverse="RelatingDataset" />
+		<DocAttribute Name="DatasetInfoForObjects" UniqueId="b516123d-f0a9-4d8d-b03f-d43496183c16" DefinedType="IfcRelAssociatesDataset" AggregationType="3" Inverse="RelatingDataset">
+			<Documentation>The dataset information with which objects are associated.</Documentation>
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Resource definition data schemas/Schemas/IfcExternalReferenceResource/Entities/IfcDatasetInformation/DocEntity.xml]: This one seems inversed. Should it not be "The dataset for which the information applies."?

File: IFC4x3/Sections/Resource definition data schemas/Schemas/IfcExternalReferenceResource/Entities/IfcDatasetReference/DocEntity.xml [hunk 2]
```diff
@@ -4,7 +4,9 @@
 		<DocAttribute Name="Description" UniqueId="ca30fa31-ac3e-4469-ae6f-26d92e6ff6b2" DefinedType="IfcText" AttributeFlags="1">
 			<Documentation>Description of the dataset reference for informational purposes.</Documentation>
 		</DocAttribute>
-		<DocAttribute Name="ReferencedDataset" UniqueId="b967d95b-2e3b-44f4-9251-a812c28ce43c" DefinedType="IfcDatasetInformation" AttributeFlags="1" />
+		<DocAttribute Name="ReferencedDataset" UniqueId="b967d95b-2e3b-44f4-9251-a812c28ce43c" DefinedType="IfcDatasetInformation" AttributeFlags="1">
+			<Documentation>The dataset that is referenced.</Documentation>
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Resource definition data schemas/Schemas/IfcExternalReferenceResource/Entities/IfcDatasetReference/DocEntity.xml]: Same as on info. Probably these two should just be switched, right?

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcGeoScienceFeatureType/Documentation.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+The _IfcGeoScienceFeatureType_ provides the type information for _IfcGeoScienceFeature_ occurrences.
+An _IfcGeoScienceFeature_ represents a geological or geotechnical feature as an interpretation of factual data such as observations, measurements and tests.
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcGeoScienceFeatureType/Documentation.md:2]: Do you want the description of IfcGeoScienceFeature on IfcGeoScienceFeatureType?

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcGeoScienceModelType/Documentation.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+The _IfcGeoScienceModelType_ provides the type information for _IfcGeoScienceModel_ occurrences.
+An _IfcGeoScienceModel_ represents a model of the geological structure as considered relevant for the project, as a base for the definition of building- and design-related geotechnical models, hydrogeological models and geo hazard models.
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcGeoScienceModelType/Documentation.md:2]: Do you want the description of IfcGeoScienceModel on IfcGeoScienceModelType?

File: IFC4x3/Sections/Resource definition data schemas/Schemas/IfcExternalReferenceResource/Entities/IfcDatasetInformation/DocEntity.xml (hunk 1 above)
Comment (larswik) [reply IFC4x3/Sections/Resource definition data schemas/Schemas/IfcExternalReferenceResource/Entities/IfcDatasetInformation/DocEntity.xml]: > This one seems inversed. Should it not be "The dataset for which the information applies."?

I think that I agree, but it is completely in line with the corresponding IfcDocumentInformation.DocumentInfoForObjects so in that case, I think that we should update both.
An alternative description could be "The objects for which the dataset/document information applies."

File: IFC4x3/Sections/Resource definition data schemas/Schemas/IfcExternalReferenceResource/Entities/IfcDatasetReference/DocEntity.xml (hunk 2 above)
Comment (larswik) [reply IFC4x3/Sections/Resource definition data schemas/Schemas/IfcExternalReferenceResource/Entities/IfcDatasetReference/DocEntity.xml]: > Same as on info. Probably these two should just be switched, right?

No, I think that this one is correct and in line with IfcDocumentReference.ReferencedDocument.

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcGeoScienceFeatureType/Documentation.md (hunk 3 above)
Comment (larswik) [reply IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcGeoScienceFeatureType/Documentation.md:2]: Maybe not. I just looked at other type objects in share infra as templates.

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcGeoScienceModelType/Documentation.md (hunk 4 above)
Comment (larswik) [reply IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcGeoScienceModelType/Documentation.md:2]: Maybe not. I just looked at other type objects in share infra  as templates.

File: IFC4x3/Sections/Resource definition data schemas/Schemas/IfcExternalReferenceResource/Entities/IfcDatasetInformation/DocEntity.xml (hunk 1 above)
Comment (larswik) [reply IFC4x3/Sections/Resource definition data schemas/Schemas/IfcExternalReferenceResource/Entities/IfcDatasetInformation/DocEntity.xml]: I updated the description in [ceafeae](https://github.com/bSI-InfraRoom/IFC-Specification/pull/869/commits/ceafeaefdfa0f366ca709c7d40619a8a7c4a5da0).
Should I also update DocumentInfoForObjects?

Comment (larswik) [reply IFC4x3/Sections/Resource definition data schemas/Schemas/IfcExternalReferenceResource/Entities/IfcDatasetInformation/DocEntity.xml]: Also updated IfcDocumentInformation.DocumentInfoForObjects in [39a07c2](https://github.com/bSI-InfraRoom/IFC-Specification/pull/869/commits/39a07c27fb24abced66e01e611f9230ab1d73764)

File: IFC4x3/Sections/Resource definition data schemas/Schemas/IfcExternalReferenceResource/Entities/IfcDatasetReference/DocEntity.xml (hunk 2 above)
Comment (larswik) [reply IFC4x3/Sections/Resource definition data schemas/Schemas/IfcExternalReferenceResource/Entities/IfcDatasetReference/DocEntity.xml]: Updated description for both IfcDatasetReference and IfcDocumentReference in [39a07c2](https://github.com/bSI-InfraRoom/IFC-Specification/pull/869/commits/39a07c27fb24abced66e01e611f9230ab1d73764)

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**